v1.0-rc.2.md

```diff
@@ -0,0 +1,88 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.0-rc.2.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.0-rc.2.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.0-rc.2.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### General Changes
+
+- [Diagnostics reporting](diagnostics-reporting.html) now includes anonymized query statistics. [#15491](https://github.com/cockroachdb/cockroach/pull/15491)
+
+### SQL Language Changes
+
+- Restrictions on schema changes in transactions with other statements have been relaxed. It is now possible to insert into a table in the same transaction that created it, provided there are no [`FOREIGN KEY`](foreign-key.html) or [`INTERLEAVE`](interleave-in-parent.html) clauses. [#15511](https://github.com/cockroachdb/cockroach/pull/15511)
+- The distributed SQL engine supports queries that internally generate rows with no columns. [#15420](https://github.com/cockroachdb/cockroach/pull/15420)
+- New variables added to the [cluster-wide settings](cluster-settings.html): `server.declined_reservation_timeout`, `server.failed_reservation_timeout`, and `server.time_until_store_dead`. [#15581](https://github.com/cockroachdb/cockroach/pull/15581)
+
+### Command-Line Interface Changes
+
+- If the new flag `--listening-url-file` is passed to [`cockroach start`](start-a-node.html), the server will write its SQL connection URL to the specified file on successful startup, in addition to printing it to `stdout`. This is particularly helpful in identifying the node's port when an unused port is assigned automatically (`--port=0`).[#15468](https://github.com/cockroachdb/cockroach/pull/15468)
+- The [`cockroach sql`](use-the-built-in-sql-client.html) shell no longer buffers all results in memory before printing them. [#15277](https://github.com/cockroachdb/cockroach/pull/15277)
+- The log file is now flushed more reliably when a node exits. [#15531](https://github.com/cockroachdb/cockroach/pull/15531)
+- The new flag `--max-offset` on the [`cockroach start`](start-a-node.html) command defines the maximum allowed clock offset for the cluster and replaces the environment variable `COCKROACH_MAX_OFFSET`. Note that the `--max-offset` value must be the same on all nodes in the cluster. In order to change it, every node in the cluster must be stopped and restarted with the new value. [#15568](https://github.com/cockroachdb/cockroach/pull/15568)
+- The `--raft-tick-interval` flag has been removed from the [`cockroach start`](start-a-node.html) command. [#15547](https://github.com/cockroachdb/cockroach/pull/15547)
+- Surface errors encountered during certificate and key load. [#15655](https://github.com/cockroachdb/cockroach/pull/15655)
```

Comment:
Flip the sentence: "Errors encountered during certificate and key load are now reported to the user in the commands where they generate."